storage/storage.go

```diff
@@ -760,22 +770,6 @@ func (storage DBStorage) GetLatestKafkaOffset() (types.KafkaOffset, error) {
 	return offset, err
 }
 
-func (storage DBStorage) getReportUpsertQuery() string {
-	if storage.dbDriverType == types.DBDriverSQLite3 {
-		return `
-			INSERT OR REPLACE INTO report(org_id, cluster, report, reported_at, last_checked_at, kafka_offset, gathered_at)
-			VALUES ($1, $2, $3, $4, $5, $6, $7)
-		`
-	}
-
-	return `
-		INSERT INTO report(org_id, cluster, report, reported_at, last_checked_at, kafka_offset, gathered_at)
-		VALUES ($1, $2, $3, $4, $5, $6, $7)
-		ON CONFLICT (cluster)
-		DO UPDATE SET org_id = $1, report = $3, reported_at = $4, last_checked_at = $5, kafka_offset = $6, gathered_at = $7
-	`
-}
-
 // GetRuleHitInsertStatement method prepares DB statement to be used to write
 // rule FQDN + rule error key into rule_hit table for given cluster_id
 func (storage DBStorage) GetRuleHitInsertStatement(rules []types.ReportItem) string {
```

Comment:
Is this related to `func (storage DBStorage) getRuleHitInsertStatement() string` ? They have the same name.

Reply:
TBH I don't know, it is something that I didn't touch in this PR. The function body looks quite different.